pipelines/jobs/configurations/jdk11u_pipeline_config.groovy

```diff
@@ -80,14 +80,16 @@ class Config11 {
                         hotspot: 'xlc13&&aix710',
                         openj9:  'xlc13&&aix715'
                 ],
-                test                : 'default'
+                test                : 'default',
+                cleanWorkspaceAfterBuild: true
         ],
 
         s390xLinux    : [
                 os                  : 'linux',
                 arch                : 's390x',
                 test                : 'default',
-                configureArgs       : '--enable-dtrace=auto'
+                configureArgs       : '--enable-dtrace=auto',
+                cleanWorkspaceAfterBuild: true
```

Comment:
Can we make this clean the buid output only? Should be safe enough since the machines have a reasonable amount of space compared to the AIX ones. (We should also aim to work towards doing the same on AIX)

Reply:
Done. Removed s390x so will use the default of BuildOutput only.